### PR TITLE
Release 49

### PR DESCRIPTION
**Pull request by tekin**
- Admin user can deactivate and activate suppliers on frameworks
- Added code to automate generation of users with late tasks list
- Updated API endpoints for replacing a return with a no-business return
- Simplified scheduling of monthly tasks
- Removed redundant depedent field validations
- Improved setup documentation

Changes: https://github.com/dxw/DataSubmissionServiceAPI/compare/release-48...release-49

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Change Log
 
+## [release-49] - 2019-02-26
+
+- Admin user can deactivate and activate suppliers on frameworks
+- Added code to automate generation of users with late tasks list
+- Updated API endpoints for replacing a return with a no-business return
+- Simplified scheduling of monthly tasks
+- Removed redundant depedent field validations
```
Comment (CristinaRO) [inline CHANGELOG.md]: Typo: "depedent"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
 # Change Log
 
+## [release-49] - 2019-02-26
+
+- Admin user can deactivate and activate suppliers on frameworks
+- Added code to automate generation of users with late tasks list
```
Comment (CristinaRO) [inline CHANGELOG.md]: Would "Added code to automate generation of the list of users with late tasks" be more readable?

File: CHANGELOG.md (hunk 1 above)
Comment (tekin) [reply CHANGELOG.md]: 👍

File: CHANGELOG.md (hunk 2 above)
Comment (tekin) [reply CHANGELOG.md]: 👍

**Pull request merged**